src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -10,6 +10,8 @@
     <Description>.NET for Apache Spark</Description>
     <PackageReleaseNotes>https://github.com/dotnet/spark/tree/master/docs/release-notes</PackageReleaseNotes>
     <PackageTags>spark;dotnet;csharp</PackageTags>
+    <!-- workaround due to including ArrayBufferWriter<T> source whilst TFM includes netstandard2.0 -->
+    <NoWarn>CS0436;$(NoWarn)</NoWarn>
```

Comment:
Why are we getting this warning? I believe it is because of the netstandard2.1 TFM (ABW exists on netstandard2.1). If so, I would only include the file on older TFMs (conditionally include the file for specific TFMs where its needed) so that we don't have to workaround this warning.

I don't think we should be ignoring such warnings.